diff --git a/types.d.ts b/types.d.ts
index 8e51dd630d79129842eec2f5b54ceec23517b360..79240195124551afaa84205e59b57d27bd77f28d 100644
--- a/types.d.ts
+++ b/types.d.ts
@@ -325,9 +325,6 @@ declare namespace BigNumber {
 
 declare class BigNumber implements BigNumber.Instance {
 
-  /** Used internally to identify a BigNumber instance. */
-  private readonly _isBigNumber: true;
-
   /** The coefficient of the value of this BigNumber, an array of base 1e14 integer numbers, or null. */
   readonly c: number[] | null;
 
